contracts/LifMarketMaker.sol

```diff
@@ -63,21 +68,25 @@ contract LifMarketMaker is Ownable {
     totalPeriods = _totalPeriods;
     foundationAddr = _foundationAddr;
     initialWei = msg.value;
-    initialBuyPrice = initialWei.div(lifToken.totalSupply());
+    initialBuyPrice = initialWei
+      .div(lifToken.totalSupply().div(PRICE_FACTOR))
+      .mul(PRICE_FACTOR);
+    initialSellPrice = initialBuyPrice.div(PRICE_FACTOR).mul(initialPriceSpread);
```

Comment:
Why do we `div(PRICE_FACTOR)`? Is it because `initialPriceSpread` is already multipled by it? If that's true, I think we should first `mul`, then `div`